secret_handlers_test.go

```diff
@@ -181,7 +190,15 @@ func createSecretRequest(t *testing.T, requestBody *m.AuthenticationCreateReques
 		},
 	)
 	c.Request().Header.Add("Content-Type", "application/json;charset=utf-8")
-	err = SecretCreate(c)
+
+	tenancy := middleware.Tenancy(SecretCreate)
+	userCatcher := middleware.UserCatcher(tenancy)
+
+	testUserId := "testUser"
+	identityHeader := testutils.IdentityHeaderForUser(testUserId)
+	c.Set("identity", identityHeader)
+
+	err = userCatcher(c)
```

Comment:
I believe it is not needed to use here middleware.Tenancy(), tests will passed without it

Reply:
Yes, I probably forgot to add test for it - so I added coverage for it,  thanks!